docs/adr/0022-flow-bit-rate-properties.md

```diff
@@ -0,0 +1,189 @@
+The variation in segment duration is assumed to be between 0.5 and 1.5 the segment_duration.
+The `SDmax` is approximately `segment_duration * 1.5` and the minimum buffer size in bytes is then approximately `max_bit_rate * 1000 * SDmax * 8`.
+
+Would a property equivalent to `minBufferTime` in DASH be useful as well?
+
+In summary, this option extends [Option 3a](#option-3a-define-segment-bit-rates) with this property:
+
+* `segment_duration` (rational, seconds): the target segment duration for segmenting the Flow media
+
+* Good, provides information about the segments that allows the bit rate information to be used to estimate receiver buffer sizes required for continuous playback (not taking network conditions into account)
+* Bad, it doesn't directly provide essence bit rate information for users of TAMS where the essence is transmitted by other means
+
+### Option 3c: Define segment bit rates with segment size
+
+SCTE 214-1 puts limits on the segment size so that they don't exceed the buffer size.
+The TAMS doesn't require media to be segmented in a way that is compatible with the restrictions applied to HLS and DASH segments.
+Some applications my optimise segments for throughput rather than taking latency into account.
+E.g. a 10 MB segment size may be the optimal segment size for an object store for maximum throughput.
+The proposal is to add `segment_size` (integer, bytes) and `max_segment_size` (integer, bytes) properties for applications that segment based on size rather than duration.
+
+In summary, this option extends [Option 3b](#option-3b-define-segment-bit-rates-with-target-segment-duration) with these properties:
+
+* `segment_size` (integer, bytes): the target segment size for segmenting the Flow media
+* `max_segment_size` (integer, bytes): the maximim segment size in TAMS for the Flow
+
+* Good, provides information about the segments optimised for size rather than duration
+* Good, provides information about the minimum buffer size required to accommodate a single segment
+* Neutral, the properties may not be used
+
+### Option 3d: Define segment bit rates with max segment duration
+
+In this option the maximum segment duration is provided explicitly by the `max_segment_duration` property rather than assuming the maximum size variation being `segment_duration * 1.5`.
+The `SDmax` is then approximately `min(segment_duration * 1.5, max_segment_duration)`
+
+In summary, this option extends [Option 3b](#option-3b-define-segment-bit-rates-with-target-segment-duration) with this property:
+
+* `max_segment_duration` (floating point, seconds): the maximum segment duration in TAMS for the Flow
+
+* Good, provides information about the segments that allows a more accurate bit rate estimate
+* Good, the calculations would not break down once segments exceed the 1.5 times the `segment_duration`.
+* Neutral, the property may not be used
+
+### Option 4: Define both essence and segment bit rate properties
```

Comment:
Should this option have it's own `Good/Neutral/Bad` list?

Reply:
I've added a list.